### DISCUSSION TITLE
Tailwind css not detect when yarn start or when production

### DISCUSSION THREAD

**Discussion by ardhichoiruddin** [vercel/next.js#22981] [Help]
### What version of Next.js are you using?

10.0.8

### What version of Node.js are you using?

12.13.0

### What browser are you using?

chrome

### What operating system are you using?

linux

### How are you deploying your application?

next start

### Describe the Bug

When I build with yarn build and after that I yarn start, tailwind css is not detected and makes my website as if it doesn't use tailwindcss. I am using tailwind version 2 together with next js. I followed the original tutorial from tailwind. https://tailwindcss.com/docs/guides/nextjs

### Expected Behavior

Hope that when it is built and at yarn start tailwind css to detect and display matches when yarn dev.

### To Reproduce

.

**Comment by franciscocobas** [comment]
Could you show us what do you have in the `tailwind.config.js` file?
